code/renderers/vue3/src/decorateStory.ts

```diff
@@ -14,56 +14,98 @@ import type { VueRenderer } from './types';
 function normalizeFunctionalComponent(options: ConcreteComponent): ComponentOptions {
   return typeof options === 'function' ? { render: options, name: options.name } : options;
 }
-
-function prepare(
-  rawStory: VueRenderer['storyResult'],
-  innerStory?: ConcreteComponent
-): Component | null {
-  const story = rawStory as ComponentOptions;
-  if (story === null) {
+/**
+ * This function takes a rawStory (as returned by the storyFn), and an innerStory (optional),
+ * which is used to render child components and returns a Component object.
+ * If an innerStory is provided, it merges the components object of the story with the components object for the innerStory.
+ *
+ * @param {VueRenderer['storyResult']} rawStory - The rawStory returned by the storyFn.
+ * @param {ConcreteComponent} innerStory - Optional innerStory used for rendering child components.
+ * @returns {Component | null} - Returns a Component object that can be rendered.
+ */
+function prepare(story: VueRenderer['storyResult'], innerStory?: Component): Component | null {
+  if (!story) {
     return null;
   }
-  if (typeof story === 'function') return story; // we don't need to wrap a functional component nor to convert it to a component options
+
+  // If story is already a function, we don't need to wrap it nor convert it
+  if (typeof story === 'function') return story;
+
+  // Normalize the functional component, and make sure inheritAttrs is set to false
+  const normalizedStory = { ...normalizeFunctionalComponent(story), inheritAttrs: false };
```

Comment:
this disable attribute fallthough  on all the  prepared stories including decorated onces,